configure.ac

```diff
@@ -201,6 +201,18 @@ AS_IF([test "x$enable_integration" = "xyes"],
        AC_SUBST([ENABLE_INTEGRATION], [$enable_integration])])
 AM_CONDITIONAL([ENABLE_INTEGRATION],[test "x$enable_integration" = "xyes"])
 #
+# asan compiler flags
+#
+AC_ARG_ENABLE([asan],
```

Comment:
Note: Might be better as a `with` configure argument so that a user could choose `(address, memory, thread, etc.)